package.json

```diff
@@ -17,7 +17,7 @@
   "dependencies": {
     "autoprefixer": "8.6.5",
     "bignumber.js": "^9.0.0",
-    "bitgo": "^11.7.1-rc.1",
+    "bitgo": "file:../BitGoJS/modules/core/bitgo-v11.5.0.tgz",
```

Comment:
note: will fix this to the right SDK version as soon as the parent PR is merged

Reply:
it's merged, you can rebase